servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStore.java

```diff
@@ -32,18 +33,18 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DynamoDBCertRecordStore.class);
     private static final Logger CERTLOGGER = LoggerFactory.getLogger("X509CertLogger");
 
-    private DynamoDB dynamoDB;
     private String tableName;
+    private DynamoDB dynamoDB;
 
     public DynamoDBCertRecordStore(AmazonDynamoDB client, final String tableName) {
-        dynamoDB = new DynamoDB(client);
+        this.dynamoDB = new DynamoDB(client);
         this.tableName = tableName;
     }
 
     @Override
     public CertRecordStoreConnection getConnection() {
         try {
-            return new DynamoDBCertRecordStoreConnection(dynamoDB, tableName);
+            return new DynamoDBCertRecordStoreConnection(dynamoDB, tableName, new DynamoDBUtils());
```

Comment:
what are we creating a new instance of DynamoDBUtils and passing here? Why can't DynamoDBCertRecordStoreConnection just maintain it's own instance of the class since it only has a couple of public methods? Any reason why those methods can't be static and then no need to create any instances?